third_party/gfootball_engine/src/systems/graphics/rendering/opengl_renderer3d.cpp

```diff
@@ -1919,14 +1929,14 @@ void LoadGLShader(GLuint shaderID, const std::string &filename) {
   GLsizei slen = 0;
   mapping.glGetShaderiv(shaderID, GL_INFO_LOG_LENGTH, &blen);
   if (blen > 1) {
-    DO_VALIDATION;
+/*    DO_VALIDATION;
```

Comment:
Commenting this because it produces the following compilation error:

>       cannot initialize a parameter of type 'GLhandleARB' (aka 'void *') with an
>       lvalue of type 'GLuint' (aka 'unsigned int')
>     mapping.glGetInfoLogARB(shaderID, blen, &slen, compiler_log);

I'm sure there is a way around it but given that is just a logging statement I just didn't want to spent to much on it ...